projects/ngrx.io/content/examples/testing-store/src/index.html

```diff
@@ -1,8 +1,19 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">
   <head>
-    <title>Angular App</title>
+    <meta charset="utf-8" />
+    <title>NgRx Tutorial</title>
+    <base href="/" />
+
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
   </head>
   <body>
-  <!-- Intentionally empty -->
+    <app-root></app-root>
   </body>
-</html>
\ No newline at end of file
+</html>
+
+<!-- 
+Use of this source code is governed by an MIT-style license that
+can be found in the LICENSE file at https://github.com/ngrx/platform
+-->
```

Comment:
Can be removed, otherwise it's added twice.

![image](https://user-images.githubusercontent.com/28659384/154814499-235898b5-6b74-4426-86c4-d4e683404825.png)